environment-dev.yml

```
@@ -2,31 +2,39 @@ name: geopandas-dev
 channels:
     - conda-forge
 dependencies:
+    - python=3.9
```

```suggestion
    - python
```

I don't think we need a pin here. Using 3.10 is fine as well.